QubitServer/src/main/java/org/labrad/qubits/jumptable/JumpTable.java

```diff
@@ -38,11 +38,11 @@ public void addEntry(String name, Data argument) {
     if (name.equals("CYCLE")) {
       List<Data> args = argument.getClusterAsList();
       Preconditions.checkArgument(args.size() == 4, "Cycle must have 4 arguments; currently has " + args.toString());
-      if (countersUsed == 3) {     // TODO: get num counters from hardware
+      if (countersUsed == 4) {     // TODO: get num counters from hardware
```

Comment:
can we factor this out into a class-level static final constant for now? The TODO about getting the value from hardware instead can go with the constant.